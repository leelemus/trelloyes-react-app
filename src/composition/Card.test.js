import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Card from './Card';

describe('Cards Commponent', () => {
    it('Card component successfully renders', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('renders Cards as expected', () => {
        const tree = renderer
            .create(<Card title="TEST TITLE" content="Test Content" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });    
});