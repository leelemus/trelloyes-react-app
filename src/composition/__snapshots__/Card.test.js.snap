// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cards Commponent renders Cards as expected 1`] = `
<div
  className="Card"
>
  <button
    onClick={[Function]}
    type="button"
  >
    delete
  </button>
  <h3>
    TEST TITLE
  </h3>
  <p>
    Test Content
  </p>
</div>
`;
